### PR TITLE
Adding retirement message for SFBO search cmdlets

### PR DESCRIPTION
**Pull request by TristanChen-msft**
Retire SFBO search cmdlets as of April 30th 2022.

### CODE REVIEW

File: skype/skype-ps/skype/Clear-CsOnlineTelephoneNumberReservation.md [hunk 1]
```diff
@@ -16,6 +16,10 @@ ms.reviewer:
 Use the `Clear-CsOnlineTelephoneNumberReservation` cmdlet to clear a reserved list of telephone numbers before they are acquired.
 The telephone numbers will then be available for search and reservation again.
 
+**Note**:
+
+As of April 30th the existing Skype for Business PowerShell Commandlets for Telephone Number Search and related activities will become end of life and will no long be available for use, the new Teams PowerShell Commandlets for Telephone Number Search and related activities are already available and can be found [here](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Clear-CsOnlineTelephoneNumberReservation.md:21]:
```suggestion
As of April 30, 2022, the existing Skype for Business PowerShell cmdlet for telephone number search and related activities will be deprecated and will no long be available for use. The new Teams PowerShell cmdlet for telephone number search and related activities are already available. For more details, see [New-CsOnlineTelephoneNumberOrder](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```

File: skype/skype-ps/skype/Get-CsOnlineDirectoryTenantNumberCities.md [hunk 2]
```diff
@@ -15,6 +15,10 @@ ms.reviewer:
 ## SYNOPSIS
 Use the Get-CsOnlineDirectoryTenantNumberCities cmdlet to retrieve the cities for which telephone numbers have been acquired by your organization.
 
+**Note**:
+
+As of April 30th the existing Skype for Business PowerShell Commandlets for Telephone Number Search and related activities will become end of life and will no long be available for use, the new Teams PowerShell Commandlets for Telephone Number Search and related activities are already available and can be found [here](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineDirectoryTenantNumberCities.md:20]:
```suggestion
As of April 30, 2022, the existing Skype for Business PowerShell cmdlet for telephone number search and related activities will be deprecated and will no long be available for use. The new Teams PowerShell cmdlet for telephone number search and related activities are already available. For more details, see [New-CsOnlineTelephoneNumberOrder](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```

File: skype/skype-ps/skype/Get-CsOnlineTelephoneNumberAvailableCount.md [hunk 3]
```diff
@@ -15,6 +15,10 @@ ms.reviewer:
 ## SYNOPSIS
 Use the Get-CsOnlineTelephoneNumberAvailableCount cmdlet to retrieve the total telephone numbers your organization is licensed to acquire.
 
+**Note**:
+
+As of April 30th the existing Skype for Business PowerShell Commandlets for Telephone Number Search and related activities will become end of life and will no long be available for use, the new Teams PowerShell Commandlets for Telephone Number Search and related activities are already available and can be found [here](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineTelephoneNumberAvailableCount.md:20]:
```suggestion
As of April 30, 2022, the existing Skype for Business PowerShell cmdlet for telephone number search and related activities will be deprecated and will no long be available for use. The new Teams PowerShell cmdlet for telephone number search and related activities are already available. For more details, see [New-CsOnlineTelephoneNumberOrder](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```

File: skype/skype-ps/skype/Get-CsOnlineTelephoneNumberInventoryAreas.md [hunk 4]
```diff
@@ -15,6 +15,10 @@ ms.reviewer:
 ## SYNOPSIS
 Use the `Get-CsOnlineTelephoneNumberInventoryAreas` cmdlet to retrieve the geographical areas where specified inventory types are supported.
 
+**Note**:
+
+As of April 30th the existing Skype for Business PowerShell Commandlets for Telephone Number Search and related activities will become end of life and will no long be available for use, the new Teams PowerShell Commandlets for Telephone Number Search and related activities are already available and can be found [here](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineTelephoneNumberInventoryAreas.md:20]:
```suggestion
As of April 30, 2022, the existing Skype for Business PowerShell cmdlet for telephone number search and related activities will be deprecated and will no long be available for use. The new Teams PowerShell cmdlet for telephone number search and related activities are already available. For more details, see [New-CsOnlineTelephoneNumberOrder](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```

File: skype/skype-ps/skype/Get-CsOnlineTelephoneNumberInventoryCities.md [hunk 5]
```diff
@@ -15,6 +15,10 @@ ms.reviewer:
 ## SYNOPSIS
 Use the `Get-CsOnlineTelephoneNumberInventoryCities` to retrieve the cities that support a given inventory type within a geographical area.
 
+**Note**:
+
+As of April 30th the existing Skype for Business PowerShell Commandlets for Telephone Number Search and related activities will become end of life and will no long be available for use, the new Teams PowerShell Commandlets for Telephone Number Search and related activities are already available and can be found [here](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineTelephoneNumberInventoryCities.md:20]:
```suggestion
As of April 30, 2022, the existing Skype for Business PowerShell cmdlet for telephone number search and related activities will be deprecated and will no long be available for use. The new Teams PowerShell cmdlet for telephone number search and related activities are already available. For more details, see [New-CsOnlineTelephoneNumberOrder](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```

File: skype/skype-ps/skype/Get-CsOnlineTelephoneNumberInventoryCountries.md [hunk 6]
```diff
@@ -15,6 +15,10 @@ ms.reviewer:
 ## SYNOPSIS
 Use the `Get-CsOnlineTelephoneNumberInventoryCountries` cmdlet to retrieve a list of countries with telephone number inventories by specified region and telephone number inventory types.
 
+**Note**:
+
+As of April 30th the existing Skype for Business PowerShell Commandlets for Telephone Number Search and related activities will become end of life and will no long be available for use, the new Teams PowerShell Commandlets for Telephone Number Search and related activities are already available and can be found [here](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineTelephoneNumberInventoryCountries.md:20]:
```suggestion
As of April 30, 2022, the existing Skype for Business PowerShell cmdlets for telephone number search and related activities will be deprecated and will no long be available for use. The new Teams PowerShell cmdlets for telephone number search and related activities are already available. For more details, see [New-CsOnlineTelephoneNumberOrder](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```

File: skype/skype-ps/skype/Get-CsOnlineTelephoneNumberInventoryRegions.md [hunk 7]
```diff
@@ -15,6 +15,10 @@ ms.reviewer:
 ## SYNOPSIS
 Use the `Get-CsOnlineTelephoneNumberInventoryRegions` cmdlet to retrieve the regions where specified inventory types are supported.
 
+**Note**:
+
+As of April 30th the existing Skype for Business PowerShell Commandlets for Telephone Number Search and related activities will become end of life and will no long be available for use, the new Teams PowerShell Commandlets for Telephone Number Search and related activities are already available and can be found [here](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineTelephoneNumberInventoryRegions.md:20]:
```suggestion
As of April 30, 2022, the existing Skype for Business PowerShell cmdlets for telephone number search and related activities will be deprecated and will no long be available for use. The new Teams PowerShell cmdlets for telephone number search and related activities are already available. For more details, see [New-CsOnlineTelephoneNumberOrder](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```

File: skype/skype-ps/skype/Get-CsOnlineTelephoneNumberInventoryTypes.md [hunk 8]
```diff
@@ -15,6 +15,10 @@ ms.reviewer:
 ## SYNOPSIS
 Use the Get-CsOnlineTelephoneNumberInventoryTypes cmdlet to retrieve the telephone number inventory types that are defined.
 
+**Note**:
+
+As of April 30th the existing Skype for Business PowerShell Commandlets for Telephone Number Search and related activities will become end of life and will no long be available for use, the new Teams PowerShell Commandlets for Telephone Number Search and related activities are already available and can be found [here](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineTelephoneNumberInventoryTypes.md:20]:
```suggestion
As of April 30, 2022, the existing Skype for Business PowerShell cmdlets for telephone number search and related activities will be deprecated and will no long be available for use. The new Teams PowerShell cmdlets for telephone number search and related activities are already available. For more details, see [New-CsOnlineTelephoneNumberOrder](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```

File: skype/skype-ps/skype/Get-CsOnlineTelephoneNumberReservationsInformation.md [hunk 9]
```diff
@@ -15,6 +15,10 @@ ms.reviewer:
 ## SYNOPSIS
 Use the Get-CsOnlineTelephoneNumberReservationsInformation to retrieve information about the total number of telephone numbers which can reserved per session, and the maximum active reservations per session.
 
+**Note**:
+
+As of April 30th the existing Skype for Business PowerShell Commandlets for Telephone Number Search and related activities will become end of life and will no long be available for use, the new Teams PowerShell Commandlets for Telephone Number Search and related activities are already available and can be found [here](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineTelephoneNumberReservationsInformation.md:20]:
```suggestion
As of April 30, 2022, the existing Skype for Business PowerShell cmdlets for telephone number search and related activities will be deprecated and will no long be available for use. The new Teams PowerShell cmdlets for telephone number search and related activities are already available. For more details, see [New-CsOnlineTelephoneNumberOrder](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```

File: skype/skype-ps/skype/Search-CsOnlineTelephoneNumberInventory.md [hunk 10]
```diff
@@ -15,6 +15,10 @@ ms.reviewer:
 ## SYNOPSIS
 Use the `Search-CsOnlineTelephoneNumberInventory` cmdlet to reserve a telephone numbers that are in inventory and available to be acquired.
 
+**Note**:
+
+As of April 30th the existing Skype for Business PowerShell Commandlets for Telephone Number Search and related activities will become end of life and will no long be available for use, the new Teams PowerShell Commandlets for Telephone Number Search and related activities are already available and can be found [here](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Search-CsOnlineTelephoneNumberInventory.md:20]:
```suggestion
As of April 30, 2022, the existing Skype for Business PowerShell cmdlets for telephone number search and related activities will be deprecated and will no long be available for use. The new Teams PowerShell cmdlets for telephone number search and related activities are already available. For more details, see [New-CsOnlineTelephoneNumberOrder](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```

File: skype/skype-ps/skype/Select-CsOnlineTelephoneNumberInventory.md [hunk 11]
```diff
@@ -16,6 +16,10 @@ ms.reviewer:
 Use the `Select-CsOnlineTelephoneNumberInventory` cmdlet to acquire a list of inventoried telephone numbers and associate them with a Business Voice Directory tenant.
 The input must be from a telephone number search operation generated by the `Search-CsOnlineTelephoneNumberInventory` cmdlet.
 
+**Note**:
+
+As of April 30th the existing Skype for Business PowerShell Commandlets for Telephone Number Search and related activities will become end of life and will no long be available for use, the new Teams PowerShell Commandlets for Telephone Number Search and related activities are already available and can be found [here](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Select-CsOnlineTelephoneNumberInventory.md:21]:
```suggestion
As of April 30, 2022, the existing Skype for Business PowerShell cmdlets for telephone number search and related activities will be deprecated and will no long be available for use. The new Teams PowerShell cmdlets for telephone number search and related activities are already available. For more details, see [New-CsOnlineTelephoneNumberOrder](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @TristanChen-msft, kindly commit the suggested changes if you agree with them. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

File: skype/skype-ps/skype/Clear-CsOnlineTelephoneNumberReservation.md [hunk 12]
```diff
@@ -16,6 +16,10 @@ ms.reviewer:
 Use the `Clear-CsOnlineTelephoneNumberReservation` cmdlet to clear a reserved list of telephone numbers before they are acquired.
 The telephone numbers will then be available for search and reservation again.
 
+**Note**:
+
+As of April 30, 2022, the existing Skype for Business PowerShell cmdlets for telephone number search and related activities will be deprecated and will no long be available for use. The new Teams PowerShell cmdlets for telephone number search and related activities are already available. For more details, see [New-CsOnlineTelephoneNumberOrder](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Clear-CsOnlineTelephoneNumberReservation.md:21]:
```suggestion
As of April 30, 2022, the existing Skype for Business PowerShell cmdlets for telephone number search and related activities will be deprecated and will no longer be available for use. The new Teams PowerShell cmdlets for telephone number search and related activities are already available. For more details, see [New-CsOnlineTelephoneNumberOrder](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```

File: skype/skype-ps/skype/Get-CsOnlineDirectoryTenantNumberCities.md [hunk 13]
```diff
@@ -15,6 +15,10 @@ ms.reviewer:
 ## SYNOPSIS
 Use the Get-CsOnlineDirectoryTenantNumberCities cmdlet to retrieve the cities for which telephone numbers have been acquired by your organization.
 
+**Note**:
+
+As of April 30, 2022, the existing Skype for Business PowerShell cmdlets for telephone number search and related activities will be deprecated and will no long be available for use. The new Teams PowerShell cmdlets for telephone number search and related activities are already available. For more details, see [New-CsOnlineTelephoneNumberOrder](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineDirectoryTenantNumberCities.md:20]:
```suggestion
As of April 30, 2022, the existing Skype for Business PowerShell cmdlets for telephone number search and related activities will be deprecated and will no longer be available for use. The new Teams PowerShell cmdlets for telephone number search and related activities are already available. For more details, see [New-CsOnlineTelephoneNumberOrder](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```

File: skype/skype-ps/skype/Get-CsOnlineTelephoneNumberAvailableCount.md [hunk 14]
```diff
@@ -15,6 +15,10 @@ ms.reviewer:
 ## SYNOPSIS
 Use the Get-CsOnlineTelephoneNumberAvailableCount cmdlet to retrieve the total telephone numbers your organization is licensed to acquire.
 
+**Note**:
+
+As of April 30, 2022, the existing Skype for Business PowerShell cmdlets for telephone number search and related activities will be deprecated and will no long be available for use. The new Teams PowerShell cmdlets for telephone number search and related activities are already available. For more details, see [New-CsOnlineTelephoneNumberOrder](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineTelephoneNumberAvailableCount.md:20]:
```suggestion
As of April 30, 2022, the existing Skype for Business PowerShell cmdlets for telephone number search and related activities will be deprecated and will no longer be available for use. The new Teams PowerShell cmdlets for telephone number search and related activities are already available. For more details, see [New-CsOnlineTelephoneNumberOrder](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```

File: skype/skype-ps/skype/Get-CsOnlineTelephoneNumberInventoryAreas.md [hunk 15]
```diff
@@ -15,6 +15,10 @@ ms.reviewer:
 ## SYNOPSIS
 Use the `Get-CsOnlineTelephoneNumberInventoryAreas` cmdlet to retrieve the geographical areas where specified inventory types are supported.
 
+**Note**:
+
+As of April 30, 2022, the existing Skype for Business PowerShell cmdlets for telephone number search and related activities will be deprecated and will no long be available for use. The new Teams PowerShell cmdlets for telephone number search and related activities are already available. For more details, see [New-CsOnlineTelephoneNumberOrder](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineTelephoneNumberInventoryAreas.md:20]:
```suggestion
As of April 30, 2022, the existing Skype for Business PowerShell cmdlets for telephone number search and related activities will be deprecated and will no longer be available for use. The new Teams PowerShell cmdlets for telephone number search and related activities are already available. For more details, see [New-CsOnlineTelephoneNumberOrder](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```

File: skype/skype-ps/skype/Get-CsOnlineTelephoneNumberInventoryCountries.md [hunk 16]
```diff
@@ -15,6 +15,10 @@ ms.reviewer:
 ## SYNOPSIS
 Use the `Get-CsOnlineTelephoneNumberInventoryCountries` cmdlet to retrieve a list of countries with telephone number inventories by specified region and telephone number inventory types.
 
+**Note**:
+
+As of April 30, 2022, the existing Skype for Business PowerShell cmdlets for telephone number search and related activities will be deprecated and will no long be available for use. The new Teams PowerShell cmdlets for telephone number search and related activities are already available. For more details, see [New-CsOnlineTelephoneNumberOrder](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineTelephoneNumberInventoryCountries.md:20]:
```suggestion
As of April 30, 2022, the existing Skype for Business PowerShell cmdlets for telephone number search and related activities will be deprecated and will no longer be available for use. The new Teams PowerShell cmdlets for telephone number search and related activities are already available. For more details, see [New-CsOnlineTelephoneNumberOrder](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```

File: skype/skype-ps/skype/Get-CsOnlineTelephoneNumberInventoryRegions.md [hunk 17]
```diff
@@ -15,6 +15,10 @@ ms.reviewer:
 ## SYNOPSIS
 Use the `Get-CsOnlineTelephoneNumberInventoryRegions` cmdlet to retrieve the regions where specified inventory types are supported.
 
+**Note**:
+
+As of April 30, 2022, the existing Skype for Business PowerShell cmdlets for telephone number search and related activities will be deprecated and will no long be available for use. The new Teams PowerShell cmdlets for telephone number search and related activities are already available. For more details, see [New-CsOnlineTelephoneNumberOrder](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineTelephoneNumberInventoryRegions.md:20]:
```suggestion
As of April 30, 2022, the existing Skype for Business PowerShell cmdlets for telephone number search and related activities will be deprecated and will no longer be available for use. The new Teams PowerShell cmdlets for telephone number search and related activities are already available. For more details, see [New-CsOnlineTelephoneNumberOrder](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```

File: skype/skype-ps/skype/Get-CsOnlineTelephoneNumberInventoryTypes.md [hunk 18]
```diff
@@ -15,6 +15,10 @@ ms.reviewer:
 ## SYNOPSIS
 Use the Get-CsOnlineTelephoneNumberInventoryTypes cmdlet to retrieve the telephone number inventory types that are defined.
 
+**Note**:
+
+As of April 30, 2022, the existing Skype for Business PowerShell cmdlets for telephone number search and related activities will be deprecated and will no long be available for use. The new Teams PowerShell cmdlets for telephone number search and related activities are already available. For more details, see [New-CsOnlineTelephoneNumberOrder](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineTelephoneNumberInventoryTypes.md:20]:
```suggestion
As of April 30, 2022, the existing Skype for Business PowerShell cmdlets for telephone number search and related activities will be deprecated and will no longer be available for use. The new Teams PowerShell cmdlets for telephone number search and related activities are already available. For more details, see [New-CsOnlineTelephoneNumberOrder](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```

File: skype/skype-ps/skype/Get-CsOnlineTelephoneNumberReservationsInformation.md [hunk 19]
```diff
@@ -15,6 +15,10 @@ ms.reviewer:
 ## SYNOPSIS
 Use the Get-CsOnlineTelephoneNumberReservationsInformation to retrieve information about the total number of telephone numbers which can reserved per session, and the maximum active reservations per session.
 
+**Note**:
+
+As of April 30, 2022, the existing Skype for Business PowerShell cmdlets for telephone number search and related activities will be deprecated and will no long be available for use. The new Teams PowerShell cmdlets for telephone number search and related activities are already available. For more details, see [New-CsOnlineTelephoneNumberOrder](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineTelephoneNumberReservationsInformation.md:20]:
```suggestion
As of April 30, 2022, the existing Skype for Business PowerShell cmdlets for telephone number search and related activities will be deprecated and will no longer be available for use. The new Teams PowerShell cmdlets for telephone number search and related activities are already available. For more details, see [New-CsOnlineTelephoneNumberOrder](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```

File: skype/skype-ps/skype/Search-CsOnlineTelephoneNumberInventory.md [hunk 20]
```diff
@@ -15,6 +15,10 @@ ms.reviewer:
 ## SYNOPSIS
 Use the `Search-CsOnlineTelephoneNumberInventory` cmdlet to reserve a telephone numbers that are in inventory and available to be acquired.
 
+**Note**:
+
+As of April 30, 2022, the existing Skype for Business PowerShell cmdlets for telephone number search and related activities will be deprecated and will no long be available for use. The new Teams PowerShell cmdlets for telephone number search and related activities are already available. For more details, see [New-CsOnlineTelephoneNumberOrder](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Search-CsOnlineTelephoneNumberInventory.md:20]:
```suggestion
As of April 30, 2022, the existing Skype for Business PowerShell cmdlets for telephone number search and related activities will be deprecated and will no longer be available for use. The new Teams PowerShell cmdlets for telephone number search and related activities are already available. For more details, see [New-CsOnlineTelephoneNumberOrder](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```

File: skype/skype-ps/skype/Select-CsOnlineTelephoneNumberInventory.md [hunk 21]
```diff
@@ -16,6 +16,10 @@ ms.reviewer:
 Use the `Select-CsOnlineTelephoneNumberInventory` cmdlet to acquire a list of inventoried telephone numbers and associate them with a Business Voice Directory tenant.
 The input must be from a telephone number search operation generated by the `Search-CsOnlineTelephoneNumberInventory` cmdlet.
 
+**Note**:
+
+As of April 30, 2022, the existing Skype for Business PowerShell cmdlets for telephone number search and related activities will be deprecated and will no long be available for use. The new Teams PowerShell cmdlets for telephone number search and related activities are already available. For more details, see [New-CsOnlineTelephoneNumberOrder](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Select-CsOnlineTelephoneNumberInventory.md:21]:
```suggestion
As of April 30, 2022, the existing Skype for Business PowerShell cmdlets for telephone number search and related activities will be deprecated and will no longer be available for use. The new Teams PowerShell cmdlets for telephone number search and related activities are already available. For more details, see [New-CsOnlineTelephoneNumberOrder](https://docs.microsoft.com/powershell/module/teams/new-csonlinetelephonenumberorder?view=teams-ps).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@TristanChen-msft, apologies, please commit these suggested changes  as well (missed them the first time around. Many thanks.

**Review by dariomws — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Review by TristanChen-msft — COMMENTED** [review]
fix typo

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**